### ISSUE TITLE
Missing gitattributes file

### ISSUE THREAD

**Issue by shantanugg** [lily-mistral/sample-java-project#4]
A gitattributes file is a simple text file that gives attributes to pathnames.
Details at https://git-scm.com/docs/gitattributes

**Comment by cdancy** [comment]
@shantanugg you're absolutely right and this was over-looked by mistake. I'll add one in later today.
